biblatex-iso690.tex

```diff
@@ -374,16 +467,55 @@ \subsubsection{Specific entry types}
 \t|type| field to specify the type of the thesis -- a localization
 term or literal string can be entered. For the list of supported
 localization terms, please refer to section 4.9.2.13 of the
-\t|biblatex| documentation. Names of the supervisor and school
-(institution) can be entered into the fields \t|supervisor| and
-\t|institution|, respectively.
+\t|biblatex| documentation. Available terms are \t|bathesis|,
+\t|mathesis|, \t|phdthesis|, and \t|candthesis|. Names of the
+supervisor and school (institution) can be entered into the fields
+\t|supervisor| and \t|institution|, respectively.
+\newline
+
+\c{Luptak2016thesis}
+\begin{ltxexample}
+@THESIS{Luptak2016thesis,
+  author       = "Lupták, Dávid",
+  title        = "Typesetting of Bibliography According to ISO 690 Norm",
+  date         = 2016,
+  howpublished = "online",
+  urldate      = "2016-06-14",
+  type         = "bathesis",
+  institution  = "Masaryk University, Faculty of Informatics",
+  location     = "Brno",
+  supervisor   = "Petr Sojka",
+  url          = "https://is.muni.cz/th/422640/fi_b/",
+}
+\end{ltxexample}
 
 \paragraph{Patent}\hfill\\
 
 The field \t|classification| is available for the respective stuff.
 For other details regarding \t|patent| type, please refer
 to the \t|biblatex| documentation.
 
+In the following example, we can see how to print out inventor of a
+patent using \t|editora| and \t|editoratype| fields, or type of patent
+using localisation string \t|patenteu|.
+\newline
+
+\c{Groll2008}
+\begin{ltxexample}
+@patent{Groll2008,
+  author         = {Clad Metals LLC Canonsburg, PA 15317 (US)},
+  title          = {Method of making a copper core five-ply composite
+                     and cooking vessel},
+  editora        = {Groll, W.A.},
+  editoratype    = {Inventor:},
+  publisher      = {Google Patents},
+  classification = {EP 1 094 937 B1},
```

Comment:
I just noticed: Classic `biblatex` would use `number` instead of `classification` here. For portability uniformity with  standard `biblatex` would be a good idea.

Reply:
I also noticed this and support to get rid of the very unique `classification` field.